### DISCUSSION TITLE
Dependency version alignment between two git projects

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#30965] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

We have two repos which are run-time dependent. Both of them producing container images.
Both of the container images are dependent on the same parent image. 
Is there a way to verify that both of them are using the same parent image version?

For context, Renovate is enabled for each repo and it is configured to process the parent image. 
My question is how to make sure both PRs are merged on the same time.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by HonkingGoose** [comment]
Maybe you're looking for this feature?

- https://github.com/renovatebot/renovate/issues/9739

**Reply by rarkins** [reply]
If both PRs are in the same repo then it sounds like you should be configuring so that you have _one_ PR doing multiple updates at once.

If they are in separate repos, that's a lot tougher. You can't control when they merge, although you could control/centralize the version in your global config or a preset using allowedVersions
